docs/scripts/platform-content-handler.js

```diff
@@ -28,7 +28,7 @@ window.addEventListener('load', () => {
 
 const initHidingLeftNavigation = () => {
     document.getElementById("leftToggler").onclick = function(event) {
-        //Events need to be prevented from bubbling since they will trigger next handler
+        // Events need to be prevented from bubbling since they will trigger next handler
```

Comment:
@jsani-radar are these docs generated? Ideally we can actually _gitignore_ this directory so these don't show up in diffs.

Reply:
Totally agree.